src/Core/ObjectRegistry.php

```diff
@@ -66,25 +66,27 @@
      * @param array $config Additional settings to use when loading the object.
      * @return mixed
      */
-    public function load($objectName, $config = [])
+    public function load($objectName, array $config = [])
```

Comment:
What about it? Please be clear just so we don't make assumptions. Other are reading and will reading those comments in the future.